rosdep/base.yaml

```diff
@@ -79,6 +79,20 @@ apr:
     slackpkg:
       packages: [apr, apr-util]
   ubuntu: [libapr1-dev, libaprutil1-dev]
+aravis:
+  debian: [libaravis-0.6-0]
+  gentoo: [media-video/aravis]
+  ubuntu:
+    '*': [libaravis-0.6-0]
+    bionic: null
+    xenial: null
```

Comment:
There are more Ubuntu versions supported by current ROS releases, but they are themselves EOL, e.g. Melodic supports Artful. But since they are EOL, we can ignore those I think. I believe nulling out bionic and xenial is sufficient.